src/locales/i18n.js

```diff
@@ -0,0 +1,10 @@
+import * as defaultTranslations from "./en/translations.json";
+import * as I18nJs from "i18n-js";
+
+function t(key){
```

Comment:
We have 2 cases here: 
- If I18n is defined in your app then use that object
- Otherwise, initialize I18nJs and use the text in `src/locales/en/translations.json`